redbox-core/redbox/models/settings.py

```diff
@@ -37,15 +37,26 @@ class ElasticCloudSettings(BaseModel):
 class Settings(BaseSettings):
     """Settings for the redbox application."""
 
-    anthropic_api_key: str | None = None
-    openai_api_key: str = "NotAKey"
-    azure_openai_api_key: str = "NotAKey"
-    azure_openai_endpoint: str | None = None
-
-    openai_api_version: str = "2023-12-01-preview"
+    # azure/gpt-35-turbo-16k
+    openai_api_version_35t: str = "2023-12-01-preview"
+    azure_openai_api_key_35t: str = "not a key"
+    azure_openai_endpoint_35t: str = "not an endpoint"
+
+    # azure/gpt-4
+    openai_api_version_4t: str = "2024-02-01"
+    azure_openai_api_key_4t: str = "not a key"
+    azure_openai_endpoint_4t: str = "not an endpoint"
+
+    # azure/gpt-4o
+    openai_api_version_4o: str = "2024-02-01"
+    azure_openai_api_key_4o: str = "not a key"
+    azure_openai_endpoint_4o: str = "not an endpoint"
+
```

Comment:
`embedding_openai_api_key` is specifically for the Embeddings ad will match one of:
* `azure_openai_api_key_35t`
* `azure_openai_api_key_4t`
* `azure_openai_api_key_4o`